data/scripts.js

```diff
@@ -2814,7 +2814,7 @@ exports.BattleScripts = {
 			item = 'DeepSeaTooth';
 		} else if (template.species === 'Spiritomb') {
 			item = 'Leftovers';
-		} else if (template.species === 'Scrafty' && counter['Status'] === 0) {
+		} else if ((template.species === 'Scrafty' || template.species === 'Escavalier' || template.species === 'Slowking' || template.species === 'Azumarill')  && counter['Status'] === 0) {
```

Comment:
Didn't see this before, oops

'Azumarill') &nbsp;&&

there are two spaces between the ``)`` and the ``&&``'s